crates/primitives/src/prune/mode.rs

```diff
@@ -17,7 +17,7 @@ pub enum PruneMode {
 #[cfg(test)]
```

Comment:
ah we only need this for tests?

otherwise can use #[default]

Reply:
yea, only for that part: https://github.com/paradigmxyz/reth/blob/73eeca0e29dda19cc9b481020c5b669fdaaff5e1/crates/storage/codecs/derive/src/compact/generator.rs#L31-L34